README.md

```diff
@@ -192,6 +192,10 @@ The `-h` flag above (shorthand of `--help`) defines that setup script must only
 - `bin/setup/xdebug`: Enable remote debugging via set of xdebug.* ini settings in .user.ini file. Please see the detailed instructions of how to configure and use it on our Wiki page [Xdebug & PHPStorm](https://github.com/mage2click/docker-magento-mutagen/wiki/Xdebug-&-PHPStorm) 
 ## Misc Info
 
+### Admin Panel
+The admin panel URL is auto-generated by the setup script. You can find the URL in the output in your terminal. The default username is `john.smith` and the default password is `password123`. You are free to change these default values in the following file:
+`compose/magento-2/bin/setup/start`
```

Comment:
I do not think anyone needs to edit any compose/magento-2/bin/setup/* files manually before the install since our main focus is to let everyone just use automated setup command with the interactive or one-line mode. I think we can create additional parameters for one-line mode to be able to specify the default admin  url / user / password during the setup w/o editing any files

Could you please remove the following sentence from your changes : "You are free to change these default values in the following file:
`compose/magento-2/bin/setup/start`" ?

Reply:
I will create new GH issue to track this idea about additional parameters fro one-line / interactive setup

Reply:
Sounds good. I'll update the changes.

Reply:
Additionally, any changes in bin/* scripts may be lost with `bin/update` or `bin/update -a` execution, because all scripts will be overwritten with newer versions.

Reply:
That's a good point. Mumble, mumble, don't edit core. 😄

Reply:
well, they are allowed if you already have project files generated once via interactive setup for example, and keeping generated files in separate git repo... but you do not need to edit anything before installing/configuring project first time via interactive/one-line setup 100%